nav/nav_goal/src/movebase_client.py

```diff
@@ -0,0 +1,83 @@
+import rospy
+import sys
+import actionlib
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from tf.transformations import quaternion_from_euler
+
+def init_node(arguments):
+    rospy.init_node('movebase_client_py')
+
+    if len(arguments) < 8:
+        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal Movebase_client x y z roll pitch yaw frame' where x is distance to move in x, y is distance to move in y, z is distance to move in z, roll, pitch, & yaw are rotations in radians, and frame is the frame of reference.")
+        rospy.signal_shutdown("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y z roll pitch yaw frame' where x is distance to move in x, y is distance to move in y, z is distance to move in z, roll, pitch, & yaw are rotations in radians, and frame is the frame of reference.")
```

Comment:
In the error message you have `rosrun nav_goal Movebase_client ....` but in the PR you have `rosrun nav_goal movebase_client.py ...`. I think you should edit this for consistency :)